README.md

```diff
@@ -134,15 +134,11 @@ images: ['/static/images/canada/mountains.jpg', '/static/images/canada/toronto.j
 
 `scripts/compose.js` can be used to easily generate a post with pre-filled front matter.
 
-The first argument is the name of the post and the second optional argument is the extension (default to .mdx)
-
-Example code to generate the post called "My First Post" in markdown format
-
+Doing
 ```
 node ./scripts/compose.js "My First Post" .md
```

Comment:
I think you need to run compose.js to have the prompts right? No need for additional arguments?

Reply:
Yes